### ISSUE TITLE
Update ganglia to the latest version (3.7.1)

### ISSUE THREAD

**Issue by chorrell** [TritonDataCenter/pkgsrc-legacy#314]
Would it be possible to get a more recent version of ganglia added? Someone was asking about it in IRC :)

Currently we just have 3.2.1 which I think dates back to 2009:

```
ganglia-monitor-core-3.1.2nb25.tgz
ganglia-webfrontend-3.1.2nb2.tgz    
```

**Comment by adriantulloch** [comment]
My company is interested in running an updated Ganglia on SmartOS, and we have a suitable patch to build ganglia-monitor-core-3.7.1 in pkgsrc. How do we go about contributing this patch to your github?

**Comment by jperkin** [comment]
Pull request would be great, we can pick it up from there.  Or if you just have a patch feel free to send that directly (maybe as a gist or something), either way is great. Thanks!

**Comment by adriantulloch** [comment]
Thanks, Jonathon. We're just testing our package and if all goes well, will supply a patch tomorrow.

**Comment by adriantulloch** [comment]
Pull request submitted https://github.com/joyent/pkgsrc/pull/322

**Comment by mamash** [comment]
Some NetBSD related patches are broken by the pull request, I'm looking at that now.

**Comment by mamash** [comment]
I spent a lot of time with the package - it was problematic in all kinds of ways as it was. I committed an update to 3.7.2 to upstream CVS trunk just now, and also threw in an update of www/ganglia-webfrontend to 3.7.1. Some of the changes implemented:
- Full SMF support
- SunOS and NetBSD build working.
- Proper setuid/setgid support, make default storage dir configurable
- Add missing buildlink3 for pcre and zlib
- Make Apache support conditional, add Nginx support (ganglia-webfrontend)

This will hit pkgsrc-2016Q1 when it's released in April.
